modules/products/rider/src/main/kotlin/com/metalbear/mirrord/products/rider/RiderPatchCommandLineExtension.kt

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("UnstableApiUsage")
```

Comment:
Probably a good idea to doc why we need to use this.

Like, say that to solve issue N we need to use API (a) which is unstable.